scripting/include/vscript.inc

```diff
@@ -221,7 +221,8 @@ methodmap VScriptExecute < Handle
 	// Creates a new handle to execute a script function
 	// 
 	// @param script           Script address to execute.
-	public native VScriptExecute(HSCRIPT script);
+	// @param scope            The script scope to execute the script inside of.
+	public native VScriptExecute(HSCRIPT script, HSCRIPT scope = view_as<HSCRIPT>(0));
```

Comment:
`HSCRIPT_RootTable` variable can be used as default scope value, to make it more clearer that root table is used as default.